sdk/core/core-client-rest/test/sendRequest.spec.ts

```diff
@@ -332,4 +332,23 @@ describe("sendRequest", () => {
       body: "test",
     });
   });
+
+  it("should have timeout option", async () => {
```

Comment:
I wasn't sure if I should add a test to testfy that timeout really works. Also can't find such test cases in the core-client.

Reply:
I think it's not strictly necessary unless there is specific logic we're worried about testing. Usually "pass-through" options are pretty straightforward vs options we have to calculate/convert.